File: Service/Admin/RMA.Service.Admin.BusinessProcessManager.Contracts/Entities/SaveWizardRequest.cs
namespace RMA.Service.Admin.BusinessProcessManager.Contracts.Entities
{
    public class SaveWizardRequest
    {
        public int WizardId { get; set; }
        public int CurrentStep { get; set; }
        public string Data { get; set; }
        public int? CustomRoutingRoleId { get; set; }
        public bool UpdateCustomRoutingRoleId { get; set; }
        public string CustomStatus { get; set; }
        public bool UpdateCustomStatus { get; set; }
        public string LockedToUser { get; set; }
        public bool UpdateLockedUser { get; set; }
        public string WizardName { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/shared-components-lib/src/lib/wizard/shared/models/save-wizard-request.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class SaveWizardRequest extends BaseClass {
    wizardId: number;
    currentStep: number;
    data: string;
    customRoutingRoleId: number;
    updateCustomRoutingRoleId: boolean;
    customStatus: string;
    updateCustomStatus: boolean;
    lockedToUser: string;
    updateLockedUser: boolean;
    wizardName: string;
}
